src/net/udp.rs

```diff
@@ -228,6 +228,14 @@ impl UdpSocket {
     /// Receives data from the socket. On success, returns the number of bytes
     /// read and the address from whence the data came.
     ///
+    /// # Notes
+    ///
+    /// On Windows, if the data is larger than the buffer specified, the buffer
+    /// is filled with the first part of the data, and recv_from returns the error
+    /// WSAEMSGSIZE(10040). The excess data is lost.
+    /// Make sure to always use a sufficiently large buffer to hold the
+    /// maximum UDP packet size, which can be up to 64 kilobytes in size.
```

Comment:
Its actually 65536 bytes, lets just use `1 << 16`.

Reply:
Same for the other notes.

Reply:
64 kilobytes is 65536 bytes. I will replace with 65536 bytes, which is the most explicit, I dont find 1 << 16 better.